src/utils/workspaceUtils.ts

```diff
@@ -66,15 +67,22 @@ export async function tryGetWorkspaceFolder(context: IActionContext): Promise<Wo
 export async function showNoWorkspacePrompt(context: IActionContext): Promise<void> {
     const noWorkspaceWarning: string = 'noWorkspaceWarning';
     const message: string = localize(noWorkspaceWarning, 'You must have a git project open to create a Static Web App.');
-    const cloneProjectMsg: MessageItem = { title: localize('cloneProject', 'Clone project from GitHub') };
-    const openExistingProjectMsg: MessageItem = { title: localize(openExistingProject, 'Open existing project') };
-    const result = await context.ui.showWarningMessage(message, { modal: true, stepName: noWorkspaceWarning }, openExistingProjectMsg, cloneProjectMsg);
+    const cloneProject = 'cloneProject';
+    const cloneProjectMsg: MessageItem = { title: localize(cloneProject, 'Clone from GitHub') };
+    const openExistingProjectMsg: MessageItem = { title: localize(openExistingProject, 'Open existing') };
+    const createFromTemplate = 'createFromTemplate';
+    const createFromTemplateMsg: MessageItem = { title: localize(createFromTemplate, 'Create from template') };
+
+    const result = await context.ui.showWarningMessage(message, { modal: true, stepName: noWorkspaceWarning }, openExistingProjectMsg, cloneProjectMsg, createFromTemplateMsg);
     if (result === cloneProjectMsg) {
         await cloneRepo(context, '');
-        context.telemetry.properties.noWorkspaceResult = 'cloneProject';
+        context.telemetry.properties.noWorkspaceResult = cloneProject;
     } else if (result === openExistingProjectMsg) {
         await openFolder(context)
         context.telemetry.properties.noWorkspaceResult = openExistingProject;
+    } else if (result === createFromTemplateMsg) {
+        await createStaticWebAppFromTemplate(context);
```

Comment:
Downside of calling `createStaticWebAppFromTemplate` here like this is the verifying workspace notification doesn't go away during the whole create process (if your entry point is the +)

I think just set the `context.isTemplate` to true here and just return undefined and go through the rest of the create process, if we refactor `createStaticWebApp` a bit.  I've left a comment about that.